app/views/two_factor_authentication/options/index.html.slim

```diff
@@ -11,18 +11,22 @@ p.mt-tiny.mb3 = @presenter.info
     fieldset.m0.p0.border-none.
       legend.mb2.serif.bold = @presenter.label
       - @presenter.options.each_with_index do |option, index|
-        label.btn-border.col-12.mb2 for="two_factor_options_form_selection_#{option.type}"
-          .radio
-            = radio_button_tag('two_factor_options_form[selection]',
+        span id="select_#{option.type}" class="#{option.html_class}"
+          label.btn-border.col-12.mb2 for="two_factor_options_form_selection_#{option.type}"
+            .radio
+              = radio_button_tag('two_factor_options_form[selection]',
                     option.type,
-                    index.zero?)
-            span.indicator.mt-tiny
-            span.blue.bold.fs-20p = option.label
-            .regular.gray-dark.fs-10p.mb-tiny = option.info
+                    index.zero?,
+                    class: option.html_class.to_s)
```

Comment:
Since this is contained within a span that might be hidden, is this class required here?

Reply:
Hidden on the radio box is part of the criteria used to decide whether or not to select another option (personal key) if webauthn is not supported and it is the first option.  Hidden on the span controls whether the option is shown or not and includes the label, etc.